products-2.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>

```

Comment:
In my final layout, I added another page which is an add on to the "all products" page to give me a clearer idea of how my final website will begin to look like.